pages/entropy/best-practices.mdx

```diff
@@ -1,5 +1,22 @@
 # Best Practices
 
+## Limit gas usage on the callback
+
+Since the entropy providers limit the gas usage on the callbacks,
+it is important to keep the callback function as simple as possible
+so that the gas usage is predictable and below the limits.
+This will ensure consistent performance and avoid potential issues with the callback.
+
+For example, if you want to use entropy to generate a random number for each player in a round of game,
+you need to make sure that the callback function works for the maximum number of players that can be in each round.
+Otherwise, the callbacks will work for some rounds with fewer players, but will fail for rounds with more players.
+
+Multiple solutions are possible to address this problem. You can store the random number received from the callback and
+either ask users to submit more transactions after the callback to continue the flow or run a background crank service
+to submit the necessary transactions.
+
+You can see the gas limits for each chain in [this](./contract-addresses) page.
```

Comment:
I suggest linking to pages using the page title, so like "The gas limits for each chain are listed on the [Contract Addresses](...) page."

that way the reader can find this info from the index if they're looking for it later.